incubator/java-openliberty/image/project/Dockerfile

```diff
@@ -6,6 +6,13 @@ RUN apt-get update && \
 
 # Copy and build the dev.appsody:java-openliberty parent pom
 COPY ./pom.xml /project/pom.xml
+
+# make usr dir for server definitions
```

Comment:
What are these for?

Reply:
I think these were there to attempt to put the usr dir someplace we could access it. I think Adam fixed this by opening permissions on the directory where liberty is installed.